recipe/meta.yaml

```diff
@@ -21,13 +21,13 @@ requirements:
     - wheel
     - python
   run:
+    - python
+    - numpy >=1.23.2
+    - pandas >=2.0.0
     - attrs >=19.3.0
     - networkx >=2.4
-    - numpy
-    - pandas >=0.25.3
-    - python
-    - tangled-up-in-unicode >=0.0.4
     - multimethod >=1.4
+    - tangled-up-in-unicode >=0.0.4
```

Comment:
This dependency was removed by the upstream source https://github.com/dylan-profiler/visions/commit/02d955e38bdf7da8badc33baac877f6d1e08b46b